quarkchain/core.py

```diff
@@ -1026,14 +1024,14 @@ def __init__(
 
     def finalize(
         self,
-        coinbase_amount_map=TokenBalanceMap(dict()),
-        coinbase_address=Address.create_empty_account()
+        coinbase_amount_map=dict(),
```

Comment:
there could be some hidden bugs if the underlying dictionary is modified later. recommended way is to use `None` as default arg and init to a new dict instance if it's `None`

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments